### DISCUSSION TITLE
Good songs recommendation

### DISCUSSION THREAD

**Discussion by selege** [NAIST-SE/AttractingContributors#545] [General]
Hi, It's good to be here.

And I am collecting good songs now. Can you just leave a song's name in my README file, then make a pull request?

Songs from any contury are fine.

This is my repository: https://github.com/selege/Good-songs/blob/main/README.md

Thank you for your cooperation.

**Comment by ani-hovhannisyan** [comment]
I contributed # 3, please contribute to my project back too:
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
Project https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git
